### DISCUSSION TITLE
munmap_chunk(): invalid pointer Error

### DISCUSSION THREAD

**Discussion by pietechsolution** [vercel/community#6795] [Help]
### Summary

Hello , 

We try to create Astro build earlier it was working with Astro V3 but after update to V4 it giving us the error.

generating static routes 
11:10:17 ▶ src/pages/about-us.astro
11:10:17   └─ /about-us/index.html (+13ms)
11:10:17 ▶ src/pages/contact-us.astro
11:10:17   └─ /contact-us/index.html (+3ms)
11:10:17 ▶ src/pages/food-delivery-app-development.astro
11:10:17   └─ /food-delivery-app-development/index.html
munmap_chunk(): invalid pointer
munmap_chunk(): invalid pointer

after the above point it stuck and runner keep running so we stop after 5-10 min. Try multiple times but not sure about , if anyone got this kind of error earlier please let me know how to fix.

### Steps to Reproduce

We have Astro project with V4 and we using astro build command

**Comment by PaulHaze** [comment]
I am also getting the exact same problem out of the blue on a new branch, nothing has changed in the dependencies from my main branch, and deployments are working fine in Cloudflare and Netlify. Astro Build works fine locally and no errors reported. 

The only error is coming Vercel, and it also seems Sharp related. I have tried both the latest version of Sharp and 0.32.3 and neither are working with Vercel. If anyone has any insight into this I would love to hear it....

**Comment by ingomc** [comment]
same problem, has it todo with astro image tools, since astro has its own image now? Locally same warnings but completes without problems, i have tried node 18 and node20... could be that vercel change the settings to node20 do default and thats throws the error?

` generating static routes 
19:22:06 ▶ src/pages/404.astro
19:22:06   └─ /404.html[astro-imagetools]  data-astro-cid-4xht4xwy  is not a valid BackgroundImage Config Option
munmap_chunk(): invalid pointer
Error: Command "npm run build" exited with SIGABRT
`

**Comment by AuJezus** [comment]
I also experience this problem

**Reply by ingomc** [reply]
i used `astro-imagetools`, i got frustrated and removed it from my project, and now my build is going again. Do you also you astro-imagetools?

[a comment by an automated account is not shown]
